devices/hive.js

```diff
@@ -113,22 +113,92 @@ module.exports = [
+            exposes.numeric('window_open_internal', ea.STATE_GET).withValueMin(0).withValueMax(4)
+                .withDescription('0=Quarantine, 1=Windows are closed, 2=Hold - Windows are maybe about to open, ' +
+                    '3=Open window detected, 4=In window open state from external but detected closed locally'),
+            exposes.binary('window_open_external', ea.ALL, true, false)
+                .withDescription('Set if the window is open or close. This setting will trigger a change in the internal ' +
+                    'window and heating demand. `false` (windows are closed) or `true` (windows are open)'),
+            exposes.numeric('day_of_week', ea.ALL).withValueMin(0).withValueMax(7)
+                .withDescription('Exercise day of week: 0=Sun...6=Sat, 7=undefined'),
+            exposes.numeric('trigger_time', ea.ALL).withValueMin(0).withValueMax(65535)
+                .withDescription('Exercise trigger time. Minutes since midnight (65535=undefined). Range 0 to 1439'),
+            exposes.numeric('algorithm_scale_factor', ea.ALL).withValueMin(1).withValueMax(10)
+                .withDescription('Scale factor of setpoint filter timeconstant ("aggressiveness" of control algorithm) '+
+                    '1= Quick ...  5=Moderate ... 10=Slow'),
+            exposes.numeric('load_estimate', ea.STATE_GET)
+                .withDescription('Load estimate on this radiator')],
         configure: async (device, coordinatorEndpoint, logger) => {
             const endpoint = device.getEndpoint(1);
-            await reporting.bind(endpoint, coordinatorEndpoint, [
-                'genBasic', 'genPowerCfg', 'genIdentify', 'genTime', 'genPollCtrl', 'hvacThermostat',
-                'hvacUserInterfaceCfg',
-            ]);
-            await reporting.thermostatTemperature(endpoint);
-            await reporting.thermostatOccupiedHeatingSetpoint(endpoint);
-            await reporting.thermostatPIHeatingDemand(endpoint);
+            const options = {manufacturerCode: 0x1246};
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genPowerCfg', 'hvacThermostat']);
+
+            // standard ZCL attributes
+            await reporting.batteryPercentageRemaining(endpoint, {min: constants.repInterval.HOUR, max: 43200, change: 1});
+            await reporting.thermostatTemperature(endpoint, {min: 0, max: constants.repInterval.MINUTES_10, change: 25});
+            await reporting.thermostatPIHeatingDemand(endpoint, {min: 0, max: constants.repInterval.MINUTES_10, change: 1});
+            await reporting.thermostatOccupiedHeatingSetpoint(endpoint, {min: 0, max: constants.repInterval.MINUTES_10, change: 25});
+
+            // danfoss attributes
+            await endpoint.configureReporting('hvacThermostat', [{
+                attribute: 'danfossMountedModeActive',
+                minimumReportInterval: constants.repInterval.MINUTE,
+                maximumReportInterval: constants.repInterval.MINUTES_10,
```

Comment:
Somehow github ate my comment, might have done it to an older commit...

Does having a max interval of 10 minutes makes sense? It will report on this at least 6 times per hour then... I doubt people will be unmounting the TRV from the valve very often. Probably having max the same as the batteryPercentage would make sense (43200). Especially since we also set reportableChange it will report as soon as the valve gets dismounted or mounted.

Reply:
@sjorge Your wild comment - you can delete it if you want since it is incomplete.
https://github.com/Koenkk/zigbee-herdsman-converters/pull/2592#commitcomment-51125580

I have changed that as per your suggestion. We didn't however have reportable change set up for danfossMountedModeActive, but I have added it to .read section.

Can you please have a look and let me know?
https://github.com/Koenkk/zigbee-herdsman-converters/pull/2592/commits/0ed0e733d0046b0828d9c6225eac8c9271989dcf

Reply:
Reporting LGTM now.